docs/topics/sealed-classes.md

```diff
@@ -80,17 +119,39 @@ you can create subclasses in any source set between the `expect` and `actual` de
 ## Sealed classes and when expression
 
 The key benefit of using sealed classes comes into play when you use them in a [`when`](control-flow.md#when-expression)
-expression. 
-If it's possible to verify that the statement covers all cases, you don't need to add an `else` clause to the statement:
+expression.
+The `when` expression, used with a sealed class, allows the Kotlin compiler to check exhaustively that all possible cases are covered. In such cases, you don't need to add an `else` clause. 
+Here's an example demonstrating this with a `when` expression:
 
 ```kotlin
+// Sealed class and its subclasses
+sealed class Error
+class FileReadError(val file: String): Error()
+class DatabaseError(val source: String): Error()
+object RuntimeError : Error()
```

Comment:
You are switching styles here: before you defined things as nested classes. Not super-important, but may puzzle people who are finding this notion for the first time.

Reply:
True! - I agree - I'll make this one nested as well to stay consistent